storage/sql.go

```diff
@@ -79,6 +72,18 @@ func newSQL(dial gorm.Dialector, level logger.LogLevel) *SQL {
 		panic(err)
 	}
 
+	conf := config.GetPlayground()
+	if config.GetPlatform() == config.Staging && conf.ForceMigration {
+		// Delete v1 tables for v2 staging
+		db.Exec("DELETE FROM users")
```

Comment:
You shouldn't write queries, that's why we have ORM to handle that for you in case we change database. Look at this https://gorm.io/docs/migration.html#Tables

Reply:
Changed to db.Migrator().DropTable("name")